ValveResourceFormat/Resource/ResourceTypes/Mesh.cs

```diff
@@ -28,10 +30,21 @@ public VBIB VBIB
 
         private VBIB cachedVBIB { get; set; }
 
+        public Attachments Attachments { get; private set; }
+
```

Comment:
This might be ok, but since the data is stored with key="name" value=attachment, I thought maybe it can be a `Dictionary<string, Attachment>`. Would have to verify if models can ever have attachments with same name (by checking existing models with decompiler -stats , or experimenting in modeldoc).

Reply:
I've replaced this with a dictionary, ModelDoc doesn't allow naming two attachments the same either. The attachments class didn't make much sense anyways after looking at it.